node_modules/oae-jitsi/lib/activity.js

```diff
@@ -0,0 +1,553 @@
+    // If the visibility has changed, we fire a visibility changed activity *instead* of an update activity
+    var activityType = null;
+    if (newMeeting.visibility === oldMeeting.visibility) {
+        activityType = MeetingsConstants.activity.ACTIVITY_MEETING_UPDATE;
+    } else {
+        activityType = MeetingsConstants.activity.ACTIVITY_MEETING_UPDATE_VISIBILITY;
+    }
+
+    var activitySeed = new ActivityModel.ActivitySeed(activityType, millis, ActivityConstants.verbs.UPDATE, actorResource, objectResource);
+    ActivityAPI.postActivity(ctx, activitySeed);
+
+});
+
+/////////////////////
+// MEETING-MESSAGE //
+/////////////////////
+
+ActivityAPI.registerActivityType(MeetingsConstants.activity.ACTIVITY_MEETING_MESSAGE, {
+    'groupBy': [{'target': true}],
+    'streams': {
+        'activity': {
+            'router': {
+                'actor': ['self'],
+                'target': ['message-contributors', 'members']
+            }
+        },
+        'notification': {
+            'router': {
+                'target': ['message-contributors', 'members']
+            }
+        },
+        'email': {
+            'router': {
+                'target': ['message-contributors', 'members']
+            }
+        },
+        'message': {
+            'transient': true,
+            'router': {
+
+                // Route the activity to the meeting
+                'target': ['self']
+            }
+        }
+    }
+});
+
+/**
+ * Post a meeting-jitsi-message activity when an user comments on a meeting
+ */
+MeetingsAPI.on(MeetingsConstants.events.CREATED_MEETING_MESSAGE, function (ctx, message, meeting) {
+
+    var millis = Date.now();
+    var actorResource = new ActivityModel.ActivitySeedResource('user', ctx.user().id, {'user': ctx.user()});
+    var objectResource = new ActivityModel.ActivitySeedResource('meeting-jitsi-message', message.id, {'meetingId': meeting.id, 'message': message});
+    var targetResource = new ActivityModel.ActivitySeedResource('meeting-jitsi', meeting.id, {'meeting-jisti': meeting});
+    var activitySeed = new ActivityModel.ActivitySeed(MeetingsConstants.activity.ACTIVITY_MEETING_MESSAGE, millis, ActivityConstants.verbs.POST, actorResource, objectResource, targetResource);
+
+    ActivityAPI.postActivity(ctx, activitySeed);
+
+});
+
+///////////////////////////
+// ACTIVITY ENTITY TYPES //
+///////////////////////////
+
+/*
+ * Produces a persistent 'meeting' activity entity
+ * @see ActivityAPI#registerActivityEntityType
+ */
+var _meetingProducer = function(resource, callback) {
+
+    var meeting = (resource.resourceData && resource.resourceData['meeting-jitsi']) ? resource.resourceData['meeting-jitsi'] : null;
+
+    // If the meeting item was fired with the resource, use it instead of fetching
+    if (meeting) return callback(null, _createPersistentMeetingActivityEntity(meeting));
+
+    MeetingsDAO.getMeeting(resource.resourceId, function(err, meeting) {
+        if (err) return callback(err);
```

Comment:
Here as well. When you run `grunt jshint` this will be one of the issues it will complain about.